modules/microshift-ingress-controller-config.adoc

```diff
@@ -91,21 +116,118 @@ ingress:
+|Describes parameters for the `Container` logging destination type. You must configure a custom logging solution that reads logs from this sidecar.
+
+|`accessLogging.destination.container.maxLength`
+|Optional configuration. The default value is `1024` bytes. Message length must be at least `480` and not greater than `8192` bytes.
+
+|`accessLogging.destination.syslog`
+|Describes parameters for the `Syslog` logging destination type. You must configure a custom Syslog instance with an endpoint that can receive Syslog messages.
+
+|`accessLogging.destination.syslog.address`
+|Required configuration when the `Syslog` destination type is set. Valid value is the IP address of the syslog endpoint that receives log messages.
+
+|`accessLogging.destination.syslog.facility`
+|Optional configuration when the `Syslog` destination type is set. Specifies the syslog facility of log messages. If this field is empty, the facility is `local1`. Otherwise, the field must specify one of the following valid syslog facilities: `kern`, `user`, `mail`, `daemon`, `auth`, `syslog`, `lpr`, `news`, `uucp`, `cron`, auth2`, `ftp`, `ntp`, `audit`, `alert`, `cron2`, `local0`, `local1`, `local2`, `local3`, `local4`, `local5`, `local6`, or `local7`.
+
+|`accessLogging.destination.syslog.maxLength`
+|Optional configuration when the `Syslog` destination type is set. The maximum length of the `Syslog` message. Message length must be at least `480` and not greater than `4096` bytes. If this field is empty, the maximum length is set to the default value of `1024` bytes.
+
+|`accessLogging.destination.syslog.port`
+|Required configuration when the `Syslog` destination type is set. The UDP port number of the syslog endpoint that receives log messages. The default value is `0`.
+
+|`httpCaptureCookies`
+|Specifies HTTP cookies that you want to capture in access logs. If the `httpCaptureCookies` field is empty, access logs do not capture the cookies. Default value is empty. Configuring `ingress.accessLogging.httpCaptureCookies` automatically enables ingress access logging. For any cookie that you want to capture, you must also set the `matchType` and `maxLength` parameters.
+
+* For example:
++
+[source,yam]
+----
+  httpCaptureCookies:
+  - matchType: Exact
+    maxLength: 128
+    name: MYCOOKIE
+----
+
+|`httpCaptureCookies.matchType`
+|Specifies whether the field name of the cookie exactly matches the capture cookie setting or is a prefix of the capture cookie setting. Valid values are `Exact` for an exact string match and `Prefix` for a string prefix match.
+
+* If you use the `Exact` setting, you must also specify a name in the `httpCaptureCookies.name` field.
+* If you use the `Prefix` setting, you must also specify a prefix in the `httpCaptureCookies.namePrefix` field. For example, the settings of `matchType: Prefix` when the `namePrefix` is "foo" captures a cookie named "foo" or "foobar" but not one named "bar". The first matching cookie is captured.
```

Comment:
IBM Style lists "foo" as a do-not-use term: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#foo